inc/classes/class-offline-form.php

```diff
@@ -56,13 +56,17 @@ protected function setup_hooks() {
 	 */
 	public function offline_form_service_worker( $scripts ) {
 
-		$scripts->register(
-			'offline-form-submit', // Handle.
-			array(
-				'src'  => array( $this, 'get_offline_form_script' ),
-				'deps' => array(), // Dependency.
-			)
-		);
+		$offline_form_sw_script = $this->get_offline_form_script();
+
+		if ( false !== $offline_form_sw_script ) {
+			$scripts->register(
+				'offline-form-submit', // Handle.
+				array(
+					'src'  => array( $this, 'get_offline_form_script' ),
```

Comment:
You can directly use `$offline_form_sw_script` as src instead of calling function again.